README.md

```diff
@@ -43,7 +43,17 @@ It provides an easy-to-use interface for developers and operators to access impo
 
 To install Kubetui, you can use the following methods:
 
-1. **Using `cargo install`:** Kubetui is available on [crates.io](https://crates.io/crates/kubetui), the official Rust package registry. Install it by running the following command:
+### **Using `brew`**
```

Comment:
Could you please modify the h3 titles in the revised section to regular text instead of bold?

```suggestion
### Using `brew`
```